src/scancode/cli.py

```diff
@@ -47,7 +47,7 @@
 from scancode_config import scancode_temp_dir
 
 from commoncode.fileutils import PATH_TYPE
-from commoncode.timeutils import time2tstamp
+from commoncode.timeutils import time2tstamp,tstamp2time
```

Comment:
Please keep each import on a separate line, but also based on my other comments `tstamp2time` is not likely needed at all.

Reply:
IMHO `time2tstamp` returns date time like this `2018-02-19T145640.979020` which is quite hard to read and understand so i converted it to `2018-02-19 14:56:40.979020` using `tstamp2time`

Reply:
IMHO It is pure luck if a datetime ends up being serialized alright.
Instead you should use this I just pushed https://github.com/nexB/scancode-toolkit/commit/179d983a222cb067a40cfdcf05de70da1ffa262f with `path_safe=False`